src/lib/provas/2019/37.json

```
@@ -0,0 +1,18 @@
+{
+    "ano": "2019",
+    "questao": "37",
+    "area": "Fundamentos",
+    "enunciado": "Dado um grafo G e um vértice de origem, qual é o algoritmo de busca que descobre \ntodos os vértices a uma distância K do vértice origem, antes de descobrir qualquer vértice a uma \ndistância K+1? ",
```

Mulher, que tanto "\n" é esse? Jesus amado

```suggestion
    "enunciado": "Dado um grafo G e um vértice de origem, qual é o algoritmo de busca que descobre todos os vértices a uma distância K do vértice origem, antes de descobrir qualquer vértice a uma distância K+1? ",
```

---

Tô rindo muito kkkkkkkk